doc-src/ch-grow_desc.rst

```diff
@@ -0,0 +1,76 @@
+Synopsis
+========
+
+::
+
+   $ ch-grow [OPTIONS] [-t TAG] [-f DOCKERFILE] CONTEXT
+
+Description
+===========
+
+Build an image named :code:`TAG` as specified in :code:`DOCKERFILE`; use
+:code:`ch-run(1)` to execute :code:`RUN` instructions. This builder is
+completely unprivileged, with no setuid/setgid/setcap helpers.
+
+:code:`ch-grow` maintains state and temporary images using normal files and
+directories. This storage directory can reside on any filesystem, and its
+location is configurable. In descending order of priority:
+
+  :code:`-s`, :code:`--storage DIR`
+    Command line option.
+
+  :code:`$CH_GROW_STORAGE`
+    Environment variable.
+
+  :code:`/var/tmp/ch-grow`
+    Default.
```

Comment:
`CH_GROW_STORAGE` has a default value in `common.bash`, however, it does not have a default value when using `ch-builder2dir`. For the latter, if `CH_GROW_STORAGE` is empty, `ch-builder2tar` will look in `/img` for the image.

```
$ ./bin/ch-builder2tar grow /tmp
building with: ch-grow
exporting and compressing
/home/jogas/upstream/bin/ch-builder2tar: line 43: cd: /img/grow:latest: No such file or directory
   0 B 0:00:00 [<=>                                                                                                                                                                         ]
-rw-rw-r--. 1 jogas jogas 20 Jun  4 15:52 /tmp/grow.tar.gz
```

Is this intended?

Reply:
No.